doc/external_docs.css

```diff
@@ -0,0 +1,19 @@
+/* unvisited link */
+a:link {
+    color: #4d76ae;
```

Comment:
Is all this going into the gh-pages? I tried [your github.io for the fork](http://9prady.github.io/arrayfire-rust) but don't see anything.